pydis_site/apps/content/resources/guides/pydis-guides/help-channel-guide.md

```diff
@@ -10,66 +10,84 @@ relevant_links:
 
-In order to claim one, simply start typing your question into one of these channels. Once your question has been posted, you have claimed this channel, and the channel will be moved down to the **Python Help: Occupied** category.
+![Available message](/static/images/content/help_channels/available_message.png)
+*This message indicates that a channel is available.*
 
-If you're unable to type into these channels, this means you're currently **on cooldown**. In order to prevent someone from claiming all the channels for themselves, **we only allow someone to claim a new help channel every 15 minutes**. However, if you close your help channel using the `!dormant` command, this cooldown is reset early.
+In order to claim one, simply ask your question in one of the available channels. Be sure to [ask good questions](../asking-good-questions) in order to give yourself the best chances of getting help!
 
-![Channel available message](/static/images/content/help_channels/available_message.png)
-*This message is always posted when a channel becomes available for use.*
+![Channel claimed embed](/static/images/content/help_channels/claimed_channel.png)
+*This messages indicates that you've claimed the channel.*
 
-## Q: For how long is the channel mine?
+At this point you will have the **Help Cooldown** role which will remain on your profile until your help channel is closed (with the `!close` command) or goes dormant due to inactivity. This ensures that users can claim only one help channel at any given time, giving everyone a chance to have their question seen.
 
-The channel is yours until it has been inactive for **30 minutes**. When this happens, we move the channel down to the **Python Help: Dormant** category, and make the channel read-only. After a while, the channel will be rotated back into **Python Help: Available** for the next question. Please try to resist the urge to continue bumping the channel so that it never gets marked as inactive. If nobody is answering your question, you should try to reformulate the question to increase your chances of getting help.
+## Frequently Asked Questions
+
+### Q: How long does my help channel stay active?
+
+The channel remains open for **30 minutes** after your last message, or 10 minutes after the last message sent by another user (whichever time comes later).
 
 ![Channel dormant message](/static/images/content/help_channels/dormant_message.png)
-*You'll see this message in your channel when the channel is marked as inactive.*
+*You'll see this message in your channel once it goes dormant.*
+### Q: No one answered my question. How come?
+
+The server has users active all over the world and all hours of the day, but some time periods are less active than others. It's also possible that the users that read your question didn't have the knowledge required to help you. If no one responded, feel free to claim another help channel a little later.
```

Comment:
This can also suggest asking in a topical channel if appropriate.

Reply:
Yes